lib/takeaway.rb

```diff
@@ -0,0 +1,60 @@
+# require_relative 'printer'
+
+class Takeaway
+    attr_reader :basket, :printer
+  def initialize(printer = Printer.new)
+    @dishes = [
+      {'Guacamole': 2},
+      {'Nachos': 5},
+      {'Tacos': 5},
+      {'Quesadillas': 6},
+      {'Taquitos': 10}, 
+      {'Tostada': 4},
+      {'Burrito': 8},
+      {'Patatas Bravas': 5}
+    ]
+    @basket = Hash.new(0)
+    @printer = printer
+  end
+
+  def print_dishes
+    @dishes
+  end
+
+  def display_menu
+    @dishes.each { |dish, price|
+      "#{dish}: #{price}"
+    }
+  end
```

Comment:
These two methods seem to be doing the same thing. I would maybe get rid of one of them - or perhaps update if the intention was for them to do different things